moveit_ros/visualization/CMakeLists.txt

```diff
@@ -54,6 +56,39 @@ add_definitions(-DQT_NO_KEYWORDS)
 # catkin_install_python(PROGRAMS scripts/moveit_joy.py DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION})
 # catkin_python_setup()
 
+set(THIS_PACKAGE_LIBRARIES
+  moveit_motion_planning_rviz_plugin
+  moveit_motion_planning_rviz_plugin_core
+  moveit_planning_scene_rviz_plugin
+  moveit_planning_scene_rviz_plugin_core
+  moveit_robot_state_rviz_plugin
+  moveit_robot_state_rviz_plugin_core
+  moveit_rviz_plugin_render_tools
+  moveit_trajectory_rviz_plugin
+  moveit_trajectory_rviz_plugin_core
+)
+
+set(THIS_PACKAGE_INCLUDE_DEPENDS
```

Comment:
This seems to be missing `moveit_ros_warehouse`. It wasn't exported before, but for some reason MTC failed to compile when not using a symlink ws.

Reply:
fixed